TurkishFilter/sections/specific.txt

```diff
@@ -892,7 +893,7 @@ saglikpersoneli.com.tr##.p_ad
 fullfilmcidayim.com##a[href="https://onwinn.com/"]
 fullfilmcidayim.com##footer ~ a[href*="bet"][title*="bet"]
 ||makroo.haber7.net/creative/makroo-hb.js
-dizimax1.com,dizifilm.pro,dizivap.*,dizimax.plus,diziday.com,dizispeed.net,dizisup.*,filmzal.*##.st_ad
+dizikorea.com,dizimax1.com,dizifilm.pro,dizivap.*,dizimax.plus,diziday.com,dizispeed.net,dizisup.*,filmzal.*##.st_ad
```

Comment:
This element has a different class in mobile view.
Please add this rule to mobile filter
`dizikorea.com##.mst_ad`